test/csync/CMakeLists.txt

```diff
@@ -1,15 +1,16 @@
 include(ECMMarkAsTest)
-include_directories(
-  ${CMAKE_CURRENT_SOURCE_DIR}
-  ${CMAKE_BINARY_DIR}
-  ${CMOCKA_INCLUDE_DIR}
-)
 
-include_directories(${CHECK_INCLUDE_DIRS})
 # create test library
 add_library(torture STATIC torture.c cmdline.c)
 target_link_libraries(torture ${CMOCKA_LIBRARIES})
 set_target_properties(torture PROPERTIES C_STANDARD 99 AUTOMOC OFF)
+target_include_directories(torture
+    PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
https://cmake.org/cmake/help/latest/variable/CMAKE_INCLUDE_CURRENT_DIR.html

And the src dir should already be included by linking to csync.